src/sentry/static/sentry/app/views/projectDetail/projectScoreCards.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import AsyncComponent from 'app/components/asyncComponent';
+import ScoreCard from 'app/components/scoreCard';
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+
+type Props = AsyncComponent['props'] & {};
+type State = AsyncComponent['state'] & {};
```

Comment:
type Props = AsyncComponent['props'];
type State = AsyncComponent['state'];

Reply:
This is WIP, there will be more stuff there once API is finished